package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.9.10",
```

Comment:
Why added material-ui when we decided not to use? Is this also needed for icons?

Reply:
Yes, material ui core is need for icons

Reply:
ok